eo-maven-plugin/src/main/java/org/eolang/maven/HashOfTag.java

```diff
@@ -52,19 +52,27 @@ public HashOfTag(final String tag) {
     /**
      * Hash of tag.
      * @return SHA of commit
-     * @throws IOException if fails
-     * @throws IllegalArgumentException if fails
      */
-    public String getHash() throws IOException, IllegalArgumentException {
-        final String link = "https://home.objectionary.com/tags.txt";
-        final InputStream ins = new URL(link).openStream();
-        final Scanner scanner = new Scanner(ins);
-        while (scanner.hasNextLine()) {
-            final String line = scanner.nextLine();
-            final String[] parts = line.split("\t");
-            if (Objects.equals(parts[1], this.tag)) {
-                Logger.info(this, "commit sha is %s", parts[0]);
-                return parts[0];
+    public String toString() {
```

Comment:
@yasamprom @yegor256
Having logic within `toString` method is not very good practice.
Especially throwing exception from within it.
Remember that it is called by default wherever string is expected - loggers, IDEs. This method should be lightweight and represent textually the object.
This class encapsulates a `tag` and `toString` should demonstrate `tag`... Something like '"Hash for TAG"'.

So I believe the first version with dedicated method was better. I would just slightly changed the name to `hash()`